### ISSUE TITLE
Unpack hed file

### ISSUE THREAD

**Issue by guitutilo** [GrzybDev/KingdomHeartsTool#13]
Version:
1.3.1

Describe the bug:
When running the script:
python -m khtool -v -u "C:\Games\Kingdom Hearts HD 1 5 and 2 5 ReMIX\Image\en\bbs_first.hed" -o output\

Generates the error reported in the image:
https://ibb.co/cNSsZcH

Desktop (please complete the following information):

OS: Windows 10
Python 3.9

**Comment by GrzybDev** [comment]
It looks like your game archive is corrupted, it's not problem with my unpack tool

**Comment by guitutilo** [comment]
You're right, I put the original files and unpacked and it worked, thanks
